src/js/controller/tracks-helper.js

```diff
@@ -24,8 +24,8 @@ define([
             return { label: label, unknownCount: unknownCount };
         },
         renderNatively: function (providerName) {
-            return (providerName === 'html5' || providerName === 'shaka' || providerName === 'caterpillar') &&
-                (browser.isChrome() || browser.isIOS() || browser.isSafari() || browser.isEdge());
+            return (providerName === 'html5' || providerName === 'shaka' || providerName === 'caterpillar' || providerName === 'hlsjs') &&
+                (browser.isChrome() || browser.isIOS() || browser.isSafari() || browser.isEdge() || (providerName === 'hlsjs' && browser.isFF()));
```

Comment:
Let's add a tech-debt ticket to simplify this. Providers that can render natively should update the model to indicate this and all others will not. That was this method isn't used in every provider that uses mixins and captions.js and view.js.